Lesson04/HW04.Birthday/Program.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
```

Comment:
I have already talked. Need to remove unused usings

Reply:
Done